src/value.c

```
@@ -288,6 +289,62 @@ const char *SIType_ToString(SIType t) {
 	}
 }
 
+void SITypes_SignatureToString(const char * fName, SIType ret, SIType *args, char *buf, size_t bufferLen) {
+	ASSERT(fName != NULL && buf != NULL);
```

cppcheck

[error] Null pointer dereference

(nullPointer)